api/src/main/java/org/openmrs/module/reporting/evaluation/querybuilder/HqlQueryBuilder.java

```diff
@@ -227,6 +227,17 @@ public HqlQueryBuilder whereInAny(String propertyName, Object... values) {
 		return this;
 	}
 
+	public HqlQueryBuilder whereNotInAny(String propertyName, Collection<?> values) {
+		if (values != null) {
+			if (values.isEmpty()) {
+				where("1=0");
```

Comment:
If the values are empty, shouldn't we return true rather than false?

Reply:
This was resolved